src/library/scala/io/Source.scala

```diff
@@ -167,6 +167,10 @@ object Source {
 
   def fromInputStream(is: InputStream)(implicit codec: Codec): BufferedSource =
     createBufferedSource(is, reset = () => fromInputStream(is)(codec), close = () => is.close())(codec)
+
+  def fromClassPath(resource: String, classLoader: ClassLoader = ClassLoader.getSystemClassLoader)(implicit codec: Codec): BufferedSource =
```

Comment:
So the method name should be `fromClassLoader`. That's probably a good point anyway, since a class path is something else. I don't know how Hawkeye missed that one. I guess you can't have too many eyes on the source. Maybe `fromResource`.